docs/ssl.md

```diff
@@ -3,7 +3,7 @@ id: ssl
 title: "Set up the SSL Certificates"
 ---
 
-Follow these instructions to configure an SSL certificate to serve an NPM registry over HTTPS.
+Follow these instructions to configure an SSL certificate to serve a NPM registry over HTTPS.
```

Comment:
Indefinite articles can be tricky, but it's always based on _how_ the proceeding word/letter is pronounced.

Hence, it should be _an_ NPM instead of _a_ NPM.

http://www.bbc.co.uk/worldservice/learningenglish/language/askaboutenglish/2010/06/100622_aae_indefinite_articles_pron.shtml

Reply:
@honzahommer I think @duncandean is right here.